test/integration/verify/controls/aws_emr_cluster.rb

```diff
@@ -0,0 +1,29 @@
+aws_emr_cluster_id = attribute(:aws_emr_cluster_id, value: '', description: 'AWS EMR cluster ID.')
+
+title 'Test single AWS EMR Cluster'
+
+control 'aws-emr-cluster-1.0' do
```

Comment:
Please also add the tests of the parameters that are involved in describe_cluster and describe_security_configuration.